common/components/forms/PositionEntryModal.jsx

```diff
@@ -7,17 +7,30 @@ import {TagDefinition} from "../utils/ProjectAPIUtils.js";
 import TagSelector from "../common/tags/TagSelector.jsx";
 import TagCategory from "../common/tags/TagCategory.jsx";
 import {PositionInfo} from "./PositionInfo.jsx";
-import url from "../utils/url.js";
-import _ from 'lodash'
+import FormValidation, {Validator} from "../forms/FormValidation.jsx";
+import formHelper from "../utils/forms.js"
+import urlHelper from "../utils/url.js";
+import isEmpty from 'lodash/isEmpty';
```

Comment:
Our convention has been to import _ and use _.isEmpty.  While your way here is perfectly valid, I feel we should keep to the existing pattern to help readability.

Reply:
Done!